bin/get_conjur_admin_key

```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/env bash
```

Comment:
I like adding this to `bin/util` as a function. Is there any use-case that this script addresses that the function doesn't? I see that it's used in a couple places (`bin/cli` and `bin/integration_tests`) but it seems like the only added value is printing the key

Reply:
Yeah, It definitely could be a function. I don't want to hold this PR back, but it could be added to a different PR before merge